sparse_strips/vello_sparse_tests/tests/renderer.rs

```diff
@@ -217,6 +217,21 @@ impl Renderer for Scene {
     }
 
     fn render_to_pixmap(&self, pixmap: &mut Pixmap, _: RenderMode) {
+        // On some platforms using `cargo test` triggers segmentation faults in wgpu when the GPU
+        // tests are run in parallel (likely related to the number of device resources being
+        // requested simultaneously). This is "fixed" by putting a mutex around this method,
+        // ensuring only one set of device resources is alive at the same time. This slows down
+        // testing when `cargo test` is used.
+        //
+        // Testing with `cargo nextest` is not meaningfully slown down. `nextest` runs each test in
```

Comment:
Hmm. `slown` is not a word I'm familiar with, and I don't think it's standard. It's borderline for inclusion in typos; if I were in charge, I'd include it, but it's the weakest one I've seen recently.

Reply:
Oops